modules/core/map/modules/google/farm_map_google.info.yml

```diff
@@ -0,0 +1,7 @@
+name: farmOS Google Map
+description: Adds Google Map layers to farmOS maps.
+type: module
+package: farmOS Maps
+core_version_requirement: ^10
```

Comment:
We'll need to change this to `^11`, but that will be caught automatically when we try to merge it into `4.x`.